core/state_transition.go

```diff
@@ -482,13 +504,16 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 
 	var gasRefund uint64
 	if !st.evm.Config.IsSystemTransaction {
-		if !rules.IsLondon {
-			// Before EIP-3529: refunds were capped to gasUsed / 2
-			gasRefund = st.refundGas(params.RefundQuotient)
-		} else {
-			// After EIP-3529: refunds are capped to gasUsed / 5
-			gasRefund = st.refundGas(params.RefundQuotientEIP3529)
+		// Compute refund counter, capped to a refund quotient.
+		gasRefund := st.calcRefund()
+		st.gas += gasRefund
```

Comment:
```suggestion
                st.gas += st.calcRefund()
```

Reply:
Opps, thank you. It's my fault, just assign new value to `gasRefund` instead of creating new `gasRefund`, because `gasRefund` is returned later. So will modify like below 
```suggestion
		gasRefund = st.calcRefund()
		st.gas += gasRefund
```